elasticapm/contrib/django/client.py

```diff
@@ -149,7 +150,12 @@ def get_data_from_request(self, request, event_type):
                         self.logger.debug("Can't capture request body: %s", str(e))
                         data = "<unavailable>"
                 if data is not None:
-                    result["body"] = data
+                    # This can be removed once we figure out a soluction to
+                    # https://github.com/elastic/apm-agent-python/issues/305
+                    if len(str(data)) > constants.KEYWORD_MAX_LENGTH:
+                        result["body"] = keyword_field(str(data))
```

Comment:
Is there a potential perf concern with `str(data)`ing twice?

Reply:
Hmm. Possibly. I'll store it in a temporary variable.